wandb/cli/cli.py

```diff
@@ -921,6 +922,24 @@ def _parse_settings(settings):
             if launch_queue.count(" ") > 1:
                 launch_queue = f"{launch_queue!r}"
 
+        scheduler_config = launch_config.get("scheduler", {})
+        scheduler_entrypoint = [
+            "wandb",
+            "scheduler",
+            "WANDB_SWEEP_ID",
+            "--queue",
+            launch_queue,
+            "--project",
+            project,
+            "--num_workers",
+            config.get("launch", {}).get("num_workers", '2'),
```

Comment:
this is from the sweep config, but we don't currently support this key. We could add it though?

Reply:
added